app/presenters/tree_node/menu/section.rb

```diff
@@ -0,0 +1,73 @@
+module TreeNode
+  module Menu
+    class Section < Node
+      set_attribute(:key) do
+        "#{@options[:node_id_prefix]}___tab_#{@object.id}"
```

Comment:
If you *can* add `tab` as a prefix for this kind of object, then it'd be better to have that, use full_ids and not override the key in any of the nodes..